tests/README.md

```diff
@@ -105,29 +105,29 @@ results summary includes the location of the complete logs, which is logged in
 the form /var/tmp/test_results/[ISO 8601 date].  A normal test run launched
 with the `zfs-tests.sh` wrapper script will look something like this:
 
-$ /usr/share/zfs/zfs-tests.sh -v -d /mnt
-
---- Configuration ---
-Runfile:         /usr/share/zfs/runfiles/linux.run
-STF_TOOLS:       /usr/share/zfs/test-runner
-STF_SUITE:       /usr/share/zfs/zfs-tests
-FILEDIR:         /mnt
-FILES:           /mnt/file-vdev0 /mnt/file-vdev1 /mnt/file-vdev2
-LOOPBACKS:       /dev/loop0 /dev/loop1 /dev/loop2 
-DISKS:           loop0 loop1 loop2 
-NUM_DISKS:       3
-FILESIZE:        4G
-Keep pool(s):    rpool
-
-/usr/share/zfs/test-runner/bin/test-runner.py  -c \
-  /usr/share/zfs/runfiles/linux.run -i /usr/share/zfs/zfs-tests
-Test: .../tests/functional/acl/posix/setup (run as root) [00:00] [PASS]
-...470 additional tests...
-Test: .../tests/functional/zvol/zvol_cli/cleanup (run as root) [00:00] [PASS]
-
-Results Summary
-PASS	 472
-
-Running Time:	00:45:09
-Percent passed:	100.0%
-Log directory:	/var/tmp/test_results/20160316T181651
+    $ /usr/share/zfs/zfs-tests.sh -v -d /mnt
+
+    --- Configuration ---
```

Comment:
Oh one other note, you may want to update this to be consistent with the current output. I think it's changed slightly.